portfolio/src/main/webapp/email.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Email</title>
+  </head>
+  <body>
+      <!-- <form action="process.php" method="POST">
+          <fieldset>
+          <legend>Contact Me </legend>
+          <label for="firstName">First Name: </label>
+          <input type="text" id="firstName" name="firstName" /> <br> <br>
+
+          <label for="lastName">Last Name: </label>
+          <input type="text" id="lastName" name="lastName" /> <br> <br>
+
+          <label for="email">Email: </label>
+          <input type="text" id="email" name="email" /> <br> <br>
+
+          <input type="submit" id="submit-button" name="submit-button" value="Contact Me"/>
+          </fieldset>
+      </form> -->
+    <form action="process.php" method="post">
+        <p>First name: <input type="text" name="firstname" /></p>
+        <p>Last name: <input type="text" name="firstname" /></p>
```

Comment:
Is there a reason you're using p tags instead of label, which would be the standard thing to do to label the inputs of a form?

Reply:
Also, the second input should have a name of "lastname".